### PR TITLE
Updated setup instructions

### PR DESCRIPTION
**Pull request by orlyohreally**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,42 @@
...
-    options: {
-      authToken: '0126997a0f359dffdf0a4200b9c229161c7610ec', // Auth token belongs to a test account with test data
-      collections: [''],
-      pages: '',
-      pageTypes: 'customer_case_study'
+  plugins: [
+    {
+      use: 'gridsome-source-buttercms',
+      options: {
+        authToken: '56655b6ca9edd2b2fd0c8b6e51148cfb119987db', // change to your own auth token
```
Comment (jakelumetta) [inline README.md]:
```suggestion
        authToken: '<your_token>', // change to your own auth token
```

File: gridsome.config.js [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
-      pageTypes: 'customer_case_study'
-    }
-  }
-  ]
-}
+  plugins: [
+    {
+      use: 'gridsome-source-buttercms',
+      options: {
+        authToken: '0126997a0f359dffdf0a4200b9c229161c7610ec', // change to your own token
```
Comment (jakelumetta) [inline gridsome.config.js]:
```suggestion
        authToken: '<your_token>', // change to your own token
```

Comment (orlyohreally) [reply gridsome.config.js]: I think this auth token is used for building demo. I have made the change but then the build failed. So I reverted to the actual token.

Comment (jakelumetta) [reply gridsome.config.js]: Ah ok thanks

**Pull request merged**